Lib/test/test_pickle.py

```diff
@@ -632,7 +632,9 @@ def test_exceptions(self):
                            ResourceWarning,
                            StopAsyncIteration,
                            RecursionError,
-                           EncodingWarning):
+                           EncodingWarning,
+                           #ExceptionGroup, # TODO: RUSTPYTHON
```

Comment:
I'd also like for the _edits_ to this particular line to live in a separate commit.

Reply:
alright :D done